src/export/CodeSystemExporter.ts

```diff
@@ -55,6 +55,18 @@ export class CodeSystemExporter {
     }
   }
 
+  private updateCount(codeSystem: CodeSystem): void {
+    // Only update the count when it isn't explicitly set, when the content is 'complete',
+    // and there is at least one concept.
+    if (
+      codeSystem.count == null &&
+      codeSystem.content === 'complete' &&
+      codeSystem.concept?.length
```

Comment:
I'm wondering if we should only do this if the `codeSystem.count` is null. Specifically what if the `codeSystem.content` is `complete`, and we can clearly count that there are 4 concepts, and the user enters 5 as a count. Should we correct that since we do know better? Since we are already inferring the count for them if they don't set it, it doesn't feel like a big stretch to expect we would fix things if they do set it and we are certain they set it wrong.

Reply:
Yeah that sounds like a good plan, that way the user has ultimate power to ignore the warning if they really feel they need to.

Reply:
OK, updated in abbfe28.